src/main/java/org/datadog/jenkins/plugins/datadog/listeners/DatadogGraphListener.java

```diff
@@ -332,7 +333,7 @@ private long getPauseDurationMillis(@Nonnull FlowNode startNode) {
 
             // In milliseconds
             return pauseDuration;
-        } catch (NullPointerException e) {
+        } catch (Exception e) {
```

Comment:
I would argue that this is making things worse, considering that the caught exception is not being logged. "Swallowing" exceptions is one of the worst things one can do: if something unexpected happens we'll be left with a generic "Unable to get the stage pause duration" error message that tells us nothing about the problem.
Suppressing an NPE is not a good thing either, but suppressing whatever problem was caught is even worse.
Given that these changes have little to do with upgrading to Java 11, my recommendation would be reverting them

Reply:
Understood, unfortunately NullPointerExceptions aren't allowed to be caught anymore, but I agree that not doing anything for **any** exception is excessive. As such, I refactored for null-checking of individual variables that might trigger a NullPointerException.